realm/realm-library/src/main/cpp/jni_util/jni_utils.cpp

```diff
@@ -65,3 +66,26 @@ void JniUtils::keep_global_ref(JavaGlobalRefByMove& ref)
     s_instance->m_global_refs.push_back(std::move(ref));
 }
 
+jobject JniUtils::to_hashmap(JNIEnv* env, std::map<std::string, std::string> map)
+{
+    jclass hash_map_class= env->FindClass("java/util/HashMap");
```

Comment:
Same as above. Looking up classes is expensive so should be minimized. If a class is used in multiple places we try to pre-load them all in `java_class_global_def.hpp`. Otherwise making the field static will make sure it is only initialized once.